src/tracer/index.ts

```diff
@@ -0,0 +1,7 @@
+export { AutoblocksTracer } from './tracer';
+export {
```

Comment:
i thought we were going to export all of this stuff from the same place as testing?